extensions/interactions/Continue/protractor.js

```diff
@@ -18,7 +18,7 @@
  */
 
 var objects = require('../../objects/protractor.js');
-var until = protractor.ExpectedConditions;
+var waitFor = require('../../,,/core/tests/protractor_utils/waitFor.js');
```

Comment:
Yup, it definitely should not be '/,,/' as per Travis

Reply:
Done. Sorry about that again!